packages/lib/BaseModel.ts

```diff
@@ -345,15 +351,24 @@ class BaseModel {
 		return this.modelSelectAll(q.sql, q.params);
 	}
 
+	public static whereIdsInSql({ ids, field = 'id', negate = false }: IdsWhereClauseOptions) {
```

Comment:
`whereIdsInSql` is responsible for generating the `ids IN (...)` clause used elsewhere.